### PR TITLE
Translation updates for 23.10 release (October, 2023)

### PR DESCRIPTION
**Pull request by akuker**
In an effort to ease the burden on our developers, I've pre-translated all of the updated messages using the paid version of PoEdit (which uses Microsoft AI in the backend). I'm hoping to get someone fluent in all of these languages to review to ensure the translations are valid.

Please review and provide any feedback/proposed changes. Also feel free to comment if you do or do not like this method of translation.

### CODE REVIEW

File: python/web/src/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -909,8 +1010,8 @@ msgstr "Info"
 
 #: src/templates/index.html:374
 #, python-format
-msgid "%(disk_space)s MiB disk space remaining on the system"
-msgstr "%(disk_space)s Mio d'espace disque restant sur le système"
+msgid "%(disk_space)s MiB disk space remaining for images"
+msgstr "%(disk_space)s Mio d'espace disque restant sur le Pi"
```
Comment (rdmark) [inline python/web/src/translations/fr/LC_MESSAGES/messages.po:1014]: This translation seems incorrect!

File: python/web/src/translations/zh/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -883,8 +965,8 @@ msgstr "信息"
 
 #: src/templates/index.html:374
 #, python-format
-msgid "%(disk_space)s MiB disk space remaining on the system"
-msgstr "系统剩余磁盘空间为%(disk_space)s MB"
+msgid "%(disk_space)s MiB disk space remaining for images"
+msgstr "在 Pi 上剩余%(disk_space)s MB 磁盘空间"
```
Comment (rdmark) [inline python/web/src/translations/zh/LC_MESSAGES/messages.po:969]: This one is wrong too...

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -120,111 +120,177 @@ msgstr "No se ha extraído ningún archivo (se omiten los archivos existentes)"
...
+msgstr ""
+"Inestabilidad potencial - Frecuencia ARM limitada - Asegure un flujo de "
+"aire/enfriamiento suficiente."
+
+#: src/return_code_mapper.py:59
+msgid ""
+"Potential instability - Currently throttled - Make sure to use a "
+"sufficient power source (2.5+ amps)."
+msgstr ""
+"Inestabilidad potencial - Actualmente estrangulado - Asegúrese de "
```
Comment (n4al) [inline python/web/src/translations/es/LC_MESSAGES/messages.po]: "Actualmente estrangulado" should be "Actualmente limitado"

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -120,111 +120,177 @@ msgstr "No se ha extraído ningún archivo (se omiten los archivos existentes)"
...
+"utilizar una fuente de alimentación suficiente (2.5+ amperios)."
+
+#: src/return_code_mapper.py:62
+msgid ""
+"Potential instability - Soft-temperature limit active - Ensure sufficient"
```
Comment (n4al) [inline python/web/src/translations/es/LC_MESSAGES/messages.po]: "Límite de temperatura suave activo" should be "Límite flexible de temperatura activo"

File: python/web/src/translations/es/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -120,111 +120,177 @@ msgstr "No se ha extraído ningún archivo (se omiten los archivos existentes)"
...
+"Inestabilidad potencial: se ha producido una limitación desde el último "
+"reinicio.  Asegúrese de utilizar una fuente de alimentación suficiente "
+"(2.5+ amperios)."
+
+#: src/return_code_mapper.py:74
+msgid ""
+"Potential instability - Soft temperature limit has occurred since last "
+"reboot.  Ensure sufficient airflow/cooling."
+msgstr ""
+"Inestabilidad potencial: se ha producido un límite de temperatura suave "
```
Comment (n4al) [inline python/web/src/translations/es/LC_MESSAGES/messages.po]: "límite de temperatura suave" should be "límite flexible de temperatura"

**Review by rdmark — APPROVED** [review]

File: python/web/src/translations/es/LC_MESSAGES/messages.po (hunk 3 above)
Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po]: Fixed

File: python/web/src/translations/es/LC_MESSAGES/messages.po (hunk 4 above)
Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po]: Fixed

File: python/web/src/translations/es/LC_MESSAGES/messages.po (hunk 5 above)
Comment (rdmark) [reply python/web/src/translations/es/LC_MESSAGES/messages.po]: Fixed

File: python/web/src/translations/fr/LC_MESSAGES/messages.po (hunk 1 above)
Comment (rdmark) [reply python/web/src/translations/fr/LC_MESSAGES/messages.po]: Fixed

File: python/web/src/translations/zh/LC_MESSAGES/messages.po (hunk 2 above)
Comment (rdmark) [reply python/web/src/translations/zh/LC_MESSAGES/messages.po]: Fixed. (Although I don't know how good the new translation is.)

**Comment by rdmark** [comment]
@cvictor Please add your commits or comments to this PR

We will wait to merge this until you have had the chance to follow up

**Comment by rdmark** [comment]
@cvictor If you don't have time today or tomorrow, we will have to revisit improvements to the German translation for future releases!

**Pull request merged**